src/app/problem/description/Description.js

```diff
@@ -0,0 +1,13 @@
+import '../ProblemEditor.css';
+import {SubtitleLine} from "../../commons/titles/TitleLine";
+
+function Description () {
+    return (
+        <div>
+            <SubtitleLine title={"Description"} />
+
+        </div>
+    )
+};
```

Comment:
if it's just a SubtitleLine then you don't need `Description` component anymore

also don't create too many unnecessary directories such as `description` where it only has one file inside

just move everything belongs to **problemEditor** to the `app/problem/edit` directory (yes, you should create `app/problem/edit` directory)